.github/workflows/build.yml

```diff
@@ -0,0 +1,56 @@
+# Initiate a github actions-initiated build
+name: Build and upload artifact on push to any branch
+on:
+  push: {}
+  workflow_dispatch: {}
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+      checks: write
+      issues: read
+      pull-requests: write
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v3
+    - name: Prepare Node
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16.x
+    # Seed the build number with last number from TeamCity.
+    - name: Update GITHUB_RUN_NUMBER
+      run: |
+        LAST_TEAMCITY_BUILD=59
+        echo GITHUB_RUN_NUMBER=$(( $GITHUB_RUN_NUMBER + $LAST_TEAMCITY_BUILD )) >> $GITHUB_ENV
```

Comment:
I wonder if it might be helpful to echo the build number here, after this line?

```
        echo Build number: $(( $GITHUB_RUN_NUMBER + $LAST_TEAMCITY_BUILD ))
```

Then at least we could see in the build log which number was chosen? And get something like this in the build log:

![image](https://user-images.githubusercontent.com/2242307/197561985-0ae8986c-0e66-4f64-be08-15b51139c509.png)

Reply:
Yes, I can do that. Be right back!